prime-router/src/test/unit_test_files/organizations.yml

```diff
@@ -1,46 +0,0 @@
----
```

Comment:
There are similar .yml and .schema files in inside src/test/csv_test_files/input/.   This has worked well to keep those tests simple, isolated, and working, but maybe there's a better place for these to go now.